src/main/java/com/exasol/releasedroid/usecases/release/ReleaseInteractor.java

```diff
@@ -61,11 +59,11 @@ private void logResults(final Report releaseReport) {
         this.reportLogger.logResults(releaseReport);
     }
 
-    private Report makeRelease(final String repositoryFullName, final List<PlatformName> platformNames) {
+    private Report makeRelease(final UserInput userInput) {
         final Report report = Report.releaseReport();
-        final Repository repository = this.repositoryGateway.getRepositoryWithDefaultBranch(repositoryFullName);
+        final Repository repository = this.repositoryGateway.getRepository(userInput);
```

Comment:
the purpose of `getRepositoryWithDefaultBranch` name is to express the intent. I suggest to keep this name.